radarr/templates/sqp/sqp-1-2160p.yml

```
@@ -15,124 +15,35 @@ radarr:
     base_url: Put your Radarr URL here
     api_key: Put your API key here
 
-# Quality Definition
-# This section defines the Quality Definitions that will be used in your Radarr instance
-# Only one set of Quality Definitions can be used per instance
-# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/quality-definition/
-    quality_definition:
-      type: sqp-streaming
-
-# Quality Profiles
-# This section creates and/or updates Quality Profile(s) in your Radarr instance
-# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/quality-profiles/
     quality_profiles:
       - name: SQP-1 (2160p)
-        reset_unmatched_scores:
-          enabled: true
-        upgrade:
-          allowed: true
-        # Choose from the following options:
-        # Use lines 38-48 for the default profile behaviour, OR
-        # comment out lines 38-48, and uncomment lines 49-59 if you
-        # prefer 2160p WEBDL releases with IMAX-E
-          until_quality: Bluray-2160p
-          until_score: 10000
-        min_format_score: 1000  # Change to 10 if you don't have access to top-tier indexers
-        score_set: sqp-1-2160p
-        quality_sort: top
-        qualities:
-          - name: Bluray-2160p
-          - name: WEB-2160p
-            qualities:
-              - WEBDL-2160p
-              - WEBRip-2160p
-          # until_quality: Bluray|WEB-2160p
-          # until_score: 10000
-        # min_format_score: 1000  # Change to 10 if you don't have access to top-tier indexers
-        # score_set: sqp-1-2160p
-        # quality_sort: top
-        # qualities:
-          # - name: Bluray|WEB-2160p
-            # qualities:
-              # - Bluray-2160p
-              # - WEBDL-2160p
-              # - WEBRip-2160p
-          # If you use SQP-1 (1080p) in another instance of Radarr, comment out lines 61-68
-          - name: Bluray|WEB-1080p
-            qualities:
-              - Bluray-1080p
-              - WEBDL-1080p
-              - WEBRip-1080p
-              - WEBDL-720p
-              - WEBRip-720p
-          - name: Bluray-720p
+        # Uncomment the below line if you don't have access to top-tier indexers
+        # min_format_score: 10
+
+    include:
+    # Choose ONE of the following four options to determine quality profile behavior.
+      - template: sqp-1-2160p-qp-default  # Default
+      # - template: sqp-1-2160p-qp-imax-e  # Prefer IMAX-E, enable IMAX-E below
+      # - template: sqp-1-2160p-4k-only-qp-default  # Default, 4K qualities only
+      # - template: sqp-1-2160p-4k-only-qp-imax-e  # Prefer IMAX-E, 4K only, enable IMAX-E below
+      # Do not comment out the next line as it enables this profile's custom formats
+      - template: sqp-1-2160p-shared
```

```suggestion
    # Choose ONE of the following four options to determine quality profile behavior.
      - template: sqp-1-2160p-qp-default  # Default
      # - template: sqp-1-2160p-qp-imax-e  # Prefer IMAX-E, enable IMAX-E below
      # - template: sqp-1-2160p-4k-only-qp-default  # Default, 4K qualities only
      # - template: sqp-1-2160p-4k-only-qp-imax-e  # Prefer IMAX-E, 4K only, enable IMAX-E below

      # Do not comment out the next line as it enables this profile's custom formats
      - template: sqp-1-2160p-shared
```

nitpick: I'd suggest introducing some vertical whitespace here to better separate things for the users.